.github/workflows/lint_ts.yml

```diff
@@ -0,0 +1,35 @@
+name: Lint on push or pull request
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: ${{ vars.TURBO_TEAM }}
+  TURBO_REMOTE_ONLY: true
```

Comment:
are these necessary? Doesn't really look like they'd be used here, and I think this is mostly a nextjs cache thing https://turborepo.com/docs/guides/ci-vendors/github-actions